bin/sl-meshctl.js

```diff
@@ -144,6 +144,12 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
+      fmt(1, 'version');
+      fmt(2, rsp.version.name, 'v%s', rsp.version.version);
```

Comment:
free-form strings can't be interacted with, I'd rather leave formatting to the client. I'll rework as key/value.